### PR TITLE
Interviewed with Julia Simplicio

### PR DESCRIPTION
**Pull request by pawan92**
Fixes https://github.com/hoodiehq/editorial/issues/75

Added Julia's interview answers. I am not aware if she has a twitter but i have linked her github profile. Any comments and suggestions are welcome.

### CODE REVIEW

**Comment by NickColley** [comment]
You can find Julia over at https://twitter.com/jlsmp

File: _posts/2016-04-20-Meet-the-hoodies-Julia-Simplicio.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+
+title: 'Meet the Hoodies: Julia Simplicio'
+author: Pawan
+layout: blog-post
+comments:
+
+---
+
+Hello all! It is time for another edition of meet the Hoodies! I had a little time to sit and chat with our very own Julia Simplicio([Twitter](https://twitter.com/jlsmp), [GitHub](https://github.com/jsimplicio)). Let’s see what she had to say:
```
Comment (jennwrites) [inline _posts/2016-04-20-Meet-the-hoodies-Julia-Simplicio.md]: Should be Meet the Hoodies 
Also it looks like maybe there's a space missing between Julia's name and her Twitter account

**Comment by jennwrites** [comment]
Once some small tweaks get made that I just noticed, lets plan on publishing this next week on Tuesday, April 26!

**Comment by pawan92** [comment]
@renrutnnej assuming you meant capitalizing 'Meet' in your comment above. in which case its fixed and ready! =)

**Pull request merged**